calculator/src/main/java/edu/nextstep/camp/calculator/HistoryViewHolder.kt

```diff
@@ -0,0 +1,24 @@
+package edu.nextstep.camp.calculator
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import edu.nextstep.camp.calculator.data.CalculationHistoryEntity
+import edu.nextstep.camp.calculator.databinding.ItemResultBinding
+
+class HistoryViewHolder(
+    private val binding: ItemResultBinding
+) : RecyclerView.ViewHolder(binding.root) {
```

Comment:
```suggestion
class HistoryViewHolder private constructor(
    private val binding: ItemResultBinding
) : RecyclerView.ViewHolder(binding.root) {
```
펙토리 메서드를 통해서 ViewHolder를 생성하고 있네요.
그렇다면 외부에서 생성자로 객체를 생성하지 못하도록 변경해보면 어떨까요?